guide/daily-spending-wallet/security.md

```diff
@@ -166,39 +158,21 @@ Bitcoin wallet users may feel hesitant about the privacy implications of your pr
 
 ### PIN protection
```

Comment:
Passphrases are generally stronger than PINs, we should mention those in this "Prevent unwanted access" section.